### ISSUE TITLE
wazap blocked my simcard but i did nothing. please help its the third line.wazapp starts to say LOGIN FAILED.I reverified the account then it said failure reason blocked   

### ISSUE THREAD

**Issue by stavon** [tgalal/wazapp#302]

**Comment by CODeRUS** [comment]
http://www.whatsapp.com/faq/general/23154266

**Comment by stavon** [comment]
ken i have the email address for whatsapp i want to send an email so that
it can activate my line again

**Comment by hgc2002** [comment]
hard to do. blocked is like end of path. i haven't been able to recover any blocked number. that's why I don't play with my own number. perhaps the solution is to be able to use another number instead of the main one form the device. this would be a nice enhancement.

**Comment by GeorgeBwanali** [comment]
This also happened to me twice, and I've tried registering using a Sim card not in my hand set. I've read the terms and conditions and I'm not In violation with it. What is causing this?

**Comment by CODeRUS** [comment]
no, this caused because using non-official software like wazapp. it's very rare case, but it's happening sometimes.

**Comment by stavon** [comment]
I have found a way to correct it. 
Contact whatsapp support team 

Sent from Samsung MobileGeorgeBwanali notifications@github.com wrote:This also happened to me twice, and I've tried registering using a Sim card not in my hand set. I've read the terms and conditions and I'm not In violation with it. What is causing this?
